app.js

```diff
@@ -1,31 +1,168 @@
 'use strict';
 
-var picOne = newPicture();
+//working with Kayla and Caleb so our code may look similar
+//var picOne = newPicture();
+
+var imgContainer = document.getElementById('img-container');
+var leftEl = document.getElementById('left');
+var centerEl = document.getElementById('center');
+var rightEl = document.getElementById('right');
 
 var img = [
   './img/bag.jpg', './img/banana.jpg', './img/bathroom.jpg', './img/boots.jpg', './img/breakfast.jpg', './img/tauntaun.jpg', './img/wine-glass.jpg', './img/bubblegum.jpg', './img/chair.jpg', './img/cthulhu.jpg', './img/dog-duck.jpg', './img/dragon.jpg', './img/unicorn.jpg', './img/usb.gif', './img/pen.jpg', './img/pet-sweep.jpg', './img/scissors.jpg', './img/shark.jpg', './img/sweep.png', './img/water-can.jpg'];
 
-var previousImgs = [];
+var totalClicks = 0;
+var clickLimits = 25;
 
-function randomNum(){
-  return Math.floor(Math.floor(Math.random() * (20 - 1 + 1) + 1));
+function product(name){
```

Comment:
Constructor functions are the one thing in javascript that gets a capitalized first letter. That's a way to indicate that this isn't a particular product, this is the template from which all products are made.